mlir/test/Dialect/Vector/canonicalize/vector-from-elements.mlir

```diff
@@ -0,0 +1,155 @@
+// RUN: mlir-opt %s -canonicalize="test-convergence" -split-input-file -allow-unregistered-dialect | FileCheck %s
+
+// This file contains some tests of folding/canonicalizing vector.from_elements
+
+///===----------------------------------------------===//
+///  Tests of `rewriteFromElementsAsSplat`
+///===----------------------------------------------===//
```

Comment:
This section was copied, right? Could you add a note in the summary so that it's easy to track the history?

Reply:
Yes, copied. I added a comment to the PR summary, I assume that's where you meant?